app/templates/partials/flash_messages/archive_service_confirmation_message.html

```diff
@@ -0,0 +1,23 @@
+{% if platform_admin %}
+      <h3 class="govuk-body govuk-!-font-weight-bold">Before you delete ‘{{service_name}}’, confirm the user: </h3>
+           <ul class="govuk-list govuk-list--bullet">
+             <li>has given the exact name of the service they want to delete</li>
+             <li>is aware they will no longer be able to use the inbound number to receive replies if there is one</li>
```

Comment:
Can we make this line conditional, so we only show it if the service has ‘Receive text messages’ enabled?

Reply:
It is possible, but since it can be quickly checked on the service settings page and it is a feature for platform admins only, I don't think it is necessary to add extra logic to deal with it.

Reply:
Happy for it to remain as is. We could test/monitor how useful platform admins find the guidance which could then inform of any changes needed.

Reply:
That approach makes sense.

In which case, this line should probably change a bit to fit with the new lead-in line.

Something like:

```suggestion
             <li>the user understands they can no longer receive text messages (if they use this feature)</li>
```